sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,6 @@
+SELECT countries.name as country,
+       count("cityId") as "#OfCities",
```

Comment:
You can also include spaces if you want: "No. of cities". That makes it hard to deal with programmatically, but to the human it's more readable.